Makefile

```diff
@@ -184,6 +184,7 @@ docker-build-controller: docker-build-go-base ## Build docker image for NeonVM c
 docker-build-runner: docker-build-go-base ## Build docker image for NeonVM runner
 	docker build \
 		--tag $(IMG_RUNNER) \
+		--build-arg TARGET_ARCH=$(TARGET_ARCH) \
```

Comment:
IIRC `neonvm-runner` doesn't yet support ARM, right? Should this be left for later?

(unless I'm misremembering, in which case, please disregard :smile:)

Reply:
probably a leftover - I originally have one branch which was split into this one and #1119